hc/api/tests/test_sendalerts.py

```diff
@@ -37,6 +36,18 @@ def test_it_handles_grace_period(self):
         check.save()
 
         # Expect no exceptions--
-        Command().handle_one(check)
+        result = Command().handle_one(check)
+        self.assertEqual(True, result, "handle_one should return True")
 
-    ### Assert when Command's handle many that when handle_many should return True
+    @patch("hc.api.management.commands.sendalerts.Command.handle_one")
+    def test_handle_many_true(self, mock):
+        """Assert when Command's handle many that when handle_many should return True"""
+        time = timezone.now() - timedelta(days=1)
+        names = ["Name %d" % n for n in range(0, 1000)]
+        for name in names:
+            check = Check(user=self.bob, name=name)
+            check.alert_after = time
+            check.status = "up"
+            check.save()
+        result = Command().handle_many()
+        self.assertEqual(result, True, "handle_many should return True")
```

Comment:
missing empty line a the end of the file

Reply:
I have rectified this @ranchow